lib/eldap/src/eldap.erl

```diff
@@ -36,6 +36,49 @@ Type definitions that are used more than once in this module:
 - **`referrals() =`** - `[Address = string()]` The contents of `Address` is
   server dependent.
 """.
+
+-type connection_info() :: #{socket := ssl:sslsocket() | gen_tcp:socket(), socket_type := tcp | ssl}.
+-type search_option() :: {base, string()} | {filter, filter()} |
+                         {scope, scope()} | {attributes, [string()]} |
+                         {deref, dereference()} | {types_only, boolean()} | {timeout, integer()}.
+
+-doc "handle() - Connection handle".
+-type handle() :: term().
+
+-doc "attribute() - `{Type = string(), Values=[string()]}`".
+-type attribute() :: {string(), [string()]}.
+
+-doc "modify_op() - See `mod_add/2`, `mod_delete/2`, `mod_replace/2`".
+-type modify_op() :: term().
+
+-doc """
+Scope of a search.
+scope() - See `baseObject/0`, `singleLevel/0`, `wholeSubtree/0`
```

Comment:
In the doc, skip "scope() - " part. It looks a bit odd in the generated documentation.
It should be sufficient with the initial text and then the "See ...".
Also, I would like to have a line break before the "See...", but I don't know how to do that in MD.

Reply:
I tried to give it a fix. Please check it. Thank you.